drivers/docker/driver.go

```diff
@@ -740,6 +740,14 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		Config: driverConfig.Logging.Config,
 	}
 
+	if hostConfig.LogConfig.Type == "" && hostConfig.LogConfig.Config == nil {
+		logger.Trace("no docker log driver provided, defaulting to json-file")
+		hostConfig.LogConfig.Type = "json-file"
+		hostConfig.LogConfig.Config = make(map[string]string)
+		hostConfig.LogConfig.Config["max-file"] = "2"
+		hostConfig.LogConfig.Config["max-size"] = "2m"
```

Comment:
You can also do the following that I find a bit more readable

```go
hostConfig.LogConfig.Config = map[string]string {
  "max-file": "2",
  "max-size": "2m"
}
```

Reply:
oof, good catch